dblookupext/esdtSupply/logsProcessor.go

```diff
@@ -192,19 +193,10 @@ func (lp *logsProcessor) shouldIgnoreEvent(event *transaction.Event) bool {
 }
 
 func (lp *logsProcessor) getESDTSupply(token string) (*SupplyESDT, error) {
-	supplyBytes, err := lp.suppliesStorer.Get([]byte(token))
+	supply, err := lp.getSupply([]byte(token))
```

Comment:
What happens when err is not ErrKeyNotFound? The method will return supply, nil. This could be nil, nil? It is ok?

Reply:
refactored